contentful/webhook-creator/README.md

```diff
@@ -0,0 +1,23 @@
+# Webhook creator
+
+Simple Javascript script that creates a Contentful webhook, ensuring it follows the same pattern across environments
+
+## Usage
+
+1. Install required packages by running `npm install`
+2. Copy `.env.example` and rename it to `.env`
+3. Fill in the required variables as described below
+4. Build the project by running `npm run build`
+5. Execute the script by running `node ./dist/create-contentful-webhook.js`
```

Comment:
Trivial but, could also be an npm script? `npm run create-webook` ?

Reply:
Done

Reply:
```suggestion
4. Create the webhook by running `npm run create-webhook`
```